core/domain/beam_job_services_test.py

```diff
@@ -82,18 +97,23 @@ def assert_domains_equal_models(self, beam_job_runs, beam_job_run_models):
         """
         self.assertEqual(len(beam_job_runs), len(beam_job_run_models))
         runs = sorted(beam_job_runs, key=lambda j: j.job_id)
-        models = sorted(beam_job_run_models, key=lambda m: m.id)
-        for i, (r, m) in enumerate(python_utils.ZIP(runs, models)):
+        # The key for sorting is defined separately because of a mypy bug.
+        # A [no-any-return] is thrown if key is defined in the sort() method
+        # instead. Reference: https://github.com/python/mypy/issues/9590.
+
```

Comment:
Maybe remove this empty line? And move it before line 100?

Reply:
Done.